osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -60,6 +62,27 @@ private void load(OverlayColourProvider colourProvider)
                             Sort = { BindTarget = Sort },
                             ShowDeleted = { BindTarget = ShowDeleted }
                         },
+                        noCommentsPlaceholder = new Container
```

Comment:
why not make this its own component and load it into content just like you load comments, rather than making it an always-present-in-hierarchy element?